server/routes/utils/reportHelper.ts

```diff
@@ -13,20 +13,25 @@
  * permissions and limitations under the License.
  */
 import puppeteer from 'puppeteer';
-import { Readable } from 'stream';
 import { v1 as uuidv1 } from 'uuid';
 
+const formats = {
+  PDF: '.pdf',
+  PNG: '.png',
+};
+
```

Comment:
Constants are defined here https://github.com/opendistro-for-elasticsearch/kibana-reports/blob/dev/server/routes/utils/constants.ts
Try to avoid duplicate code.

Reply:
good catch, will change